native/src/main/java/org/ballerinalang/redis/connection/RedisConnectionCommandExecutor.java

```diff
@@ -65,9 +78,9 @@ public String echo(String message) throws RedisConnectorException {
     }
 
     public <K> String ping() throws RedisConnectorException {
-        RedisCommands<K, String> redisCommands = null;
+        BaseRedisCommands<K, String> redisCommands = null;
         try {
-            redisCommands = (RedisCommands<K, String>) connManager.getConnectionCommandConnection();
+            redisCommands = (BaseRedisCommands<K, String>) connManager.getConnectionCommandConnection();
```

Comment:
Same as above, if this is to follow a pattern with the other methods, it would be fine to go this way I guess.

Reply:
https://github.com/ballerina-platform/module-ballerinax-redis/pull/200#discussion_r1469318859